src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,8 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService user = new UserService();
+        return user.findByEmail(email) != null
+                && user.findByEmail(email).getPassword().equals(password);
```

Comment:
It is better to call method only once, not twice
```suggestion
	User user = userService.findByEmail(email)
        return user != null && user.getPassword().equals(password);
```